### PR TITLE
WIP: Document how to install insights client and upload file (#1584)

### LINKED ISSUE #1584: Upload QPC report to file upload service (no QPC code involved)
* Figure out how to:
    * Setup insights client locally to develop (e.g. install).  Document results in README.
    * Configure insights client to authenticate with insights services.  Document results.
    * From the CLI upload a QPC report

### PR DESCRIPTION
**Pull request by myersCody**
Closes #1584

### CODE REVIEW

File: docs/source/insights_upload_configuration.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Installing the Insights Client
+------------------------------
+The Insights Client requires the insights-core which is why we coupled these repositories together inside of an insights directory. To work with the Insights Client, begin by cloning the repository::
```
Comment (abaiken) [inline docs/source/insights_upload_configuration.rst]: I would change this section to the following:

To work with the Insights Client, we must also install the Insights Core. To begin, create the insights directory and clone the following repositories::

File: docs/source/insights_upload_configuration.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We recommend that you isolate your development work by using a virtual environment. Run the following command to set up a virtual environment::
+
+    pipenv shell
+
+Accessing the Insights Development Branches
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For QPC to access the Insights Client locally we need to checkout branches that are still in development.
+
+    cd insights-core
```
Comment (abaiken) [inline docs/source/insights_upload_configuration.rst]: I'm not sure why, but if you actually view this, it is not rendering correctly. It is lumping all commands onto the same line

File: docs/source/insights_upload_configuration.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    git fetch origin platform-upload && git checkout platform-upload
+    cd ../insights-client
+    git fetch origin os-x-test && git checkout os-x-test
+
+Edit the Insights Client Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You will need modify the Insights Client Configuration in order to be authorized to upload.
+
+    vim etc/insights-client.conf
```
Comment (abaiken) [inline docs/source/insights_upload_configuration.rst]: Same comment as above for this section of commands

File: docs/source/insights_upload_configuration.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^
+After configuration is setup, you will need to build the insights client::
+
+    sudo sh lay-the-eggs-osx.sh
+
+Upload Command:
+^^^^^^^^^^^^^^^
+To upload a QPC report file using the Insight Clients you will need to run the following command.
+
+    sudo EGG=/etc/insights-client/rpm.egg BYPASS_GPG=True insights-client --no-gpg --payload=test.tar.gz --content-type=application/vnd.redhat.qpc.test+tgz
```
Comment (abaiken) [inline docs/source/insights_upload_configuration.rst]: it looks like this command is not rendering correctly either

**Comment by myersCody** [comment]
Moved to the new QPC repository: https://github.com/quipucords/qpc/pull/13

**Pull request closed without merging**